### PR TITLE
Pass breadcrumb context through to Future

### PR DESCRIPTION
**Pull request by nikz**
Passes breadcrumbs through to the Future when `send_in_background` is enabled so that they are correctly attached to the error report.

### CODE REVIEW

File: lib/raygun.rb [hunk 1]
```diff
@@ -34,6 +34,9 @@ class << self
     # Configuration Object (instance of Raygun::Configuration)
     attr_writer :configuration
 
+    # List of futures that are currently running
+    @@active_futures = []
```
Comment (sumitramanga) [inline lib/raygun.rb:38]: Just trying to figure out what a `future` is 🤔

**Review by sumitramanga — APPROVED** [review]
Code LGTM!

**Comment by TSMMark** [comment]
This is exciting, when can we expect a gem release?

**Comment by nikz** [comment]
@TSMMark we are targeting a prerelease version in the next couple of days, but want to let that soak in some real apps for a decent amount of time before a full release.

**Pull request merged**